WebApplicationMVC/wwwroot/SingleView/index.html

```diff
@@ -476,7 +480,7 @@
             // create a button in minimize taskbar
             let abbreviatedTitle = (newFrame.title.length > 15) ? `${newFrame.title.substring(0, 12)}...` : newFrame.title;
             $('#minimizeBar').append(`<button onclick="$('#${newFrame.id}').show();" style="margin-top:5px; background-color: #f3f3f3"
-                    class="button button3" id="btn_${newFrame.id}"><i style="color:black" class="${newFrame.iconClass}"></i></button>`)
+                        class="button button3" id="btn_${newFrame.id}"><i style="color:black" class="${newFrame.iconClass}"></i></button>`)
```

Comment:
shouldn't leave the tab change. it is not necessary but becomes a part of the PR code review.

Reply:
Sure.